Bio/phenotype/phen_micro.py

```diff
@@ -182,8 +183,9 @@ def _is_well(self, obj):
         """
         # Value should be of WellRecord type
         if not isinstance(obj, WellRecord):
-            raise ValueError("A WellRecord type object is needed as value" +
-                             " (got %s)" % type(obj))
+            raise ValueError(
+                "A WellRecord type object is needed as value" + " (got %s)" % type(obj)
```

Comment:
We can remove the plus and combine these strings.

(The original code had a redundant plus)